### PR TITLE
Classic Warrior

### PR DESCRIPTION
**Pull request by Ac1dBomb**
### Description

Learning as I go,  stupid syntax errors.  Might finally be fixed

### Testing

- Test report URL: `/report/...`
- Screenshot(s):

### CODE REVIEW

**Comment by Ac1dBomb** [comment]
Im stuck now,  Syntax should be right.  Its telling me something about redeclaring name part.  Let me know ill fix it up and get this right.  Sorry tried pull request bunch of different times wasnt sure if I could repull through sameone but I closed the others.

File: src/CONTRIBUTORS.ts [hunk 1]
```diff
@@ -113,6 +119,12 @@ export const Iskalla: Contributor = {
   github: 'Iskalla',
   avatar: avatar('iskalla-avatar.jpg'),
 };
+export const Pilsung: Contributor = {
+    nickname: 'Pilsung',
+    github: 'Ac1dBomb',
+    discord: 'Pilsung#5366',
+    avatar: avatar('pilsung-avatar.jpg'),
+};
```
Comment (jazminite) [inline src/CONTRIBUTORS.ts]: Remove this entry, you have it in twice

File: src/common/SPELLS/classic/warrior.ts [hunk 2]
```diff
@@ -1,7 +1,262 @@
 import { spellIndexableList } from '../Spell';
 
 const spells = spellIndexableList({
-  VIGILANCE: {
+  // --------
+  // SHARED
+  // --------
+ 
+Battle_Shout: {
```
Comment (jazminite) [inline src/common/SPELLS/classic/warrior.ts]: All object keys need to be in all CAPS to follow style format.
So this one would be changed to `BATTLE_SHOUT`.
name values use title case

File: src/common/SPELLS/classic/warrior.ts [hunk 3]
```diff
@@ -1,7 +1,262 @@
...
+    id: 47436,
+    name: 'Battle Shout (Rank 9)',
```
Comment (jazminite) [inline src/common/SPELLS/classic/warrior.ts]: Names should not have `(Rank #)`.

File: src/common/SPELLS/classic/warrior.ts [hunk 4]
```diff
@@ -1,7 +1,262 @@
...
+    icon: 'ability_warrior_battleshout', 
+   },
```
Comment (jazminite) [inline src/common/SPELLS/classic/warrior.ts]: Each spell with multiple ranks needs a `lowRanks` attribute with the lower rank ids in an array. For example:
```suggestion
    lowRanks: [2048, 25289, 11551, 11550, 11549, 6192, 5242, 6673],
   },
```

**Review by jazminite — CHANGES_REQUESTED** [review]

File: src/common/SPELLS/classic/warrior.ts [hunk 5]
```diff
@@ -1,10 +1,557 @@
-import { spellIndexableList } from '../Spell';
+/**
+ * All Warrior abilities except talents go in here. You can also put a talent in here if you want to override something imported in the `./talents` folder, but that should be extremely rare.
+ * You need to do this manually, usually an easy way to do this is by opening a WCL report and clicking the icons of spells to open the relevant Wowhead pages, here you can get the icon name by clicking the icon, copy the name of the spell and the ID is in the URL.
+ * You can access these entries like other entries in the spells files by importing `common/SPELLS` and using the assigned property on the SPELLS object. Please try to avoid abbreviating properties.
+ */
+
+import { spellIndexableList } from './Spell';
```
Comment (jazminite) [inline src/common/SPELLS/classic/warrior.ts]:
```suggestion
import { spellIndexableList } from '../Spell';
```

**Pull request closed without merging**